components/homme/src/theta-l/prim_state_mod.F90

```diff
@@ -387,9 +389,10 @@ subroutine prim_printstate(elem, tl,hybrid,hvcoord,nets,nete)
     Mass = Mass2*scale
 
     if(hybrid%masterthread) then
-       write(iulog,100) "u     = ",umin_p,umax_p,usum_p
-       write(iulog,100) "v     = ",vmin_p,vmax_p,vsum_p
-       write(iulog,100) "w     = ",wmin_p,wmax_p,wsum_p
+       write(iulog,108) "u     = ",umin_p,"     ",umax_p,"     ",usum_p
+       write(iulog,108) "v     = ",vmin_p,"     ",vmax_p,"     ",vsum_p
+       write(iulog,109) "w     = ",wmin_local(1),"(",nint(wmin_local(2)),")",&
```

Comment:
Maybe put a space between the real value and the parenthetical integer.

Reply:
done.